admin_manual/configuration_server/antivirus_configuration.rst

```diff
@@ -5,211 +5,199 @@ Virus Scanner Support
 
    netstat -a|grep clam
    unix 2 [ ACC ] STREAM LISTENING 15857 /var/run/clamav/clamd.ctl
 
   .. figure:: ../images/antivirus-daemon-socket.png
 
-The ``Stream Length`` value sets the number of bytes read in one pass.
-10485760 bytes, or ten megabytes, is the default. This value should be 
-no larger than the PHP ``memory_limit`` settings, or physical memory if 
-``memory_limit`` is set to -1 (no limit).
+The ``Stream Length`` value sets the number of bytes to read in one pass.
+10485760 bytes, or ten megabytes, is the default. 
+This value should be no larger than the PHP ``memory_limit`` settings or physical memory if ``memory_limit`` is set to -1 (no limit).
 
-``Action for infected files found while scanning`` gives you the choice of
-logging any alerts without deleting the files, or immediately deleting
-infected files.
+``Action for infected files found while scanning`` gives you the choice of logging any alerts without deleting the files or immediately deleting infected files.
 
 Daemon
 ^^^^^^
 
-For the Daemon option, you need the hostname or IP address of the remote
-server running ClamAV and the server's port number.
+In this mode, ClamAV runs on a different server. This is a good option for ownCloud servers with high volumes of file uploads.
+For the Daemon option, you need the hostname or IP address of the remote server running ClamAV and the server's port number.
 
   .. figure:: ../images/antivirus-daemon-socket.png
 
 Executable
 ^^^^^^^^^^
 
-The Executable option requires the path to ``clamscan``, which is the
-interactive ClamAV scanning command. ownCloud should find it automatically.
+In this mode, ClamAV runs on the same server as the ownCloud installation, and the ``clamscan`` command only runs when a file is uploaded. 
+``clamscan`` is slow and not always reliable for on-demand usage; it is better to use one of the daemon modes.
+
+This option requires the path to ``clamscan``, which is the interactive ClamAV scanning command. 
+ownCloud should find it automatically.
 
   .. figure:: ../images/antivirus-executable.png
 
-When you are satisfied with how ClamAV is operating, you might want to go
-back and change all of your logging to less verbose levels.
+When you are satisfied with how ClamAV is operating, you might want to go back and change all of your logging to less verbose levels.
 
 Rule Configuration
 ^^^^^^^^^^^^^^^^^^
 
-ownCloud provides the ability to customize how ownCloud reacts to the
-response provided by an anti-virus scan. To do so, under `Admin -> Antivirus
-Configuration -> Advanced`, which you can see in the screenshot below, you can
-view and change the existing rules. You can also add new ones. 
+ownCloud provides the ability to customize how it reacts to the response given by an anti-virus scan. 
+To do so, under `Admin -> Antivirus Configuration -> Advanced`, which you can see in the screenshot below, you can view and change the existing rules. 
+You can also add new ones. 
```

Comment:
@butonic 
>If admins use ClamAV do they have to customize the rules?

They don't. Default ruleset for clamAv is populated automatically.
`Scanner exit status` rules are used to handle errors when ClamAv is run in CLI mode while
`Scanner output` rules are used in daemon/socket mode.

Exit status is basically an exit code returned by any binary (not necessary ClamAv)
Daemon output is parsed by regexp.

IIRC the rules are always checked in order is always infected/error/clean order. In cases there are no matched ruled the status would be `Unknown` and a warning is logged.